src/main/java/jdbc/dao/UserDaoImpl.java

```diff
@@ -1,13 +1,13 @@
 package jdbc.dao;
 
 import jdbc.MyException;
-import jdbc.model.LoginToken;
-import jdbc.model.User;
+import jdbc.model.*;
```

Comment:
@Dimitriy1 we are not using * import

Reply:
https://mate-academy.github.io/style-guides/java/java.html#s3.3.1-wildcard-imports

Reply:
I agree with It